src/simtools/model/model_parameter.py

```diff
@@ -495,10 +497,10 @@ def write_sim_telarray_config_file(self, add_model=None):
 
     @property
     def config_file_directory(self):
-        """Directory for configure files. Configure, if necessary."""
+        """Directory for configuration files. Configure if not yet set."""
         if self._config_file_directory is None:
             self._set_config_file_directory_and_name()
-        return self._config_file_directory or self._set_config_file_directory_and_name()
+        return self._config_file_directory
```

Comment:
I thought the opposite, i.e., that 

```
return self._config_file_directory or self._set_config_file_directory_and_name()
```

would cover both cases and you won't need 

```
if self._config_file_directory is None:
            self._set_config_file_directory_and_name()
```

But it doesn't matter, we can keep it like this.